### PR TITLE
This will be a series of small scale and low risk changes for improving startup

### PR DESCRIPTION
**Pull request by ManfredKarrer**

### CODE REVIEW

File: src/main/java/bisq/seednode/SeedNodeMain.java [hunk 1]
```diff
@@ -17,181 +17,56 @@
...
-            System.out.println();
-            parser.printHelpOn(System.out);
-            System.exit(EXIT_FAILURE);
-            return;
-        }
-        BisqEnvironment bisqEnvironment = getBisqEnvironment(options);
+        if (BisqExecutable.setupInitialOptionParser(args)) {
+            // For some reason the JavaFX launch process results in us losing the thread context class loader: reset it.
+            // In order to work around a bug in JavaFX 8u25 and below, you must include the following code as the first line of your realMain method:
+            Thread.currentThread().setContextClassLoader(SeedNodeMain.class.getClassLoader());
```
Comment (blabno) [inline src/main/java/bisq/seednode/SeedNodeMain.java]: Is this still needed? That was supposed to be javafx hack.

**Review by blabno — APPROVED** [review]
Apart from that one question:

Tested ACK

File: src/main/java/bisq/seednode/SeedNodeMain.java [hunk 2]
```diff
@@ -17,181 +17,56 @@
...
 
-        // For some reason the JavaFX launch process results in us losing the thread context class loader: reset it.
-        // In order to work around a bug in JavaFX 8u25 and below, you must include the following code as the first line of your realMain method:
-        Thread.currentThread().setContextClassLoader(SeedNodeMain.class.getClassLoader());
+    @Override
+    protected void doExecute(OptionSet options) {
+        super.doExecute(options);
 
-        new SeedNodeMain().execute(args);
+        checkMemory(bisqEnvironment, seedNode);
```
Comment (blabno) [inline src/main/java/bisq/seednode/SeedNodeMain.java]: `doExecute()` from parent class will block forever with `keepRunning()`.

Comment (ManfredKarrer) [reply src/main/java/bisq/seednode/SeedNodeMain.java]: Ah good catch ;-). Will push a fix commit in a minute...

File: src/main/java/bisq/seednode/SeedNodeMain.java (hunk 1 above)
Comment (ManfredKarrer) [reply src/main/java/bisq/seednode/SeedNodeMain.java]: I am not sure. I took that over from the Lighthouse project. Would require some investigation.

**Pull request merged**